tools/run-dev.py

```diff
@@ -325,18 +325,25 @@ def shutdown_handler(*args: Any, **kwargs: Any) -> None:
     else:
         io_loop.stop()
 
-# log which services/ports will be started
-print("Starting Zulip services on ports: web proxy: {},".format(proxy_port),
-      "Django: {}, Tornado: {}, Thumbor: {}".format(django_port, tornado_port, thumbor_port),
-      end='')
-if options.test:
-    print("")  # no webpack for --test
-else:
-    print(", webpack: {}".format(webpack_port))
+def print_listeners() -> None:
+    print("\nZulip services will listen on ports:")
+    ports = [
+        (proxy_port, CYAN + 'web proxy' + ENDC),
+        (django_port, 'Django'),
+        (tornado_port, 'Tornado'),
+    ]
+
+    if not options.test:
+        ports.append((webpack_port, 'webpack'))
+
+    ports.append((thumbor_port, 'Thumbor'))
 
-print("".join((WARNING,
-               "Note: only port {} is exposed to the host in a Vagrant environment.".format(
-                   proxy_port), ENDC)))
+    for port, label in ports:
+        print(f'   {port}: {label}')
+    print()
+
+    proxy_warning = f"Only the proxy port ({proxy_port}) is exposed."
```

Comment:
Ooh, we can use fstrings now! :).  Probably worth a posting advertising this.

Reply:
I'll do that now.